requirements.txt

```diff
@@ -27,36 +29,47 @@ django-ses==0.8.14
 django-summernote==0.8.11.6
 djangorestframework==3.12.2
 docutils==0.15.2
+drf-yasg==1.20.0
 future==0.18.2
 gunicorn==19.9.0
 hiredis==1.0.0
 hyperlink==19.0.0
 idna==2.8
 importlib-metadata==1.6.0
 incremental==17.5.0
+inflection==0.5.1
+itypes==1.2.0
+Jinja2==2.11.2
 jmespath==0.10.0
 kombu==4.6.8
 Markdown==3.1.1
+MarkupSafe==1.1.1
 msgpack==0.6.1
 mysqlclient==1.4.2.post1
+packaging==20.8
 Pillow==6.0.0
+pkg-resources==0.0.0
```

Comment:
```suggestion
```
Remove this line. There is no matching distribution.
Turns out its bug crept in by Ubuntu. <a href="https://github.com/pypa/pip/issues/4022">Read here </a>to learn more about it.

Reply:
Yeah, I forgot that, Thanks ma'am!